src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -125,9 +139,56 @@ public boolean hasNote() {
     public void periodic() {
         publish();
 
-        // set the motor powers to be the value appropriate for this mode
-        motor.set(mode.power);
-        centeringMotor.set(mode.centeringPower);
```

Comment:
This should still be here. We should set the motor every frame.

Reply:
Why? The motor powers do not need to be continually set to the same value.